modules/ROOT/pages/dataweave-cookbook-java-methods.adoc

```diff
@@ -102,8 +103,22 @@ public class MyClass {
 }
 ----
 
-The DataWeave example below first imports the "MyClass" class, then creates a new instance of it and calls its instance variable "foo". Note that even though the object has a method "getFoo()", it can't be called via DataWeave.
+To create the `MyClass` example in a Studio project:
+
+. Create a project for the class in Studio by clicking *New* -> *Mule Project*
+and providing a name, such as *my-app*, and clicking *Finish*.
```

Comment:
Since this is a series, I suggest: "...by clicking *New* -> *Mule Project*,
providing a name, such as *my-app*, and clicking *Finish*."